### ISSUE TITLE
"Cannot read property 'removeEventListener' of null" on context switch before chart finished loading

### ISSUE THREAD

**Issue by OberstK** [apexcharts/vue-apexcharts#275]
# Bug report

They only reliable way I found to reproduce this, was creating a Vue App that enables a quick switch away from the loading chart while its not yet finished rendering. This provokes the error most of the time. I am not 100% sure if this related to apexcharts.js itself or the vue-apexcharts lib.

The error I get on a context switch is the following:

```
Uncaught (in promise) TypeError: Cannot read property 'removeEventListener' of null
    at eval (apexcharts.min.js?3d1e:14)
    at Array.forEach (<anonymous>)
    at t.value (apexcharts.min.js?3d1e:14)
    at t.value (apexcharts.min.js?3d1e:14)
    at eval (apexcharts.min.js?3d1e:14)
    at new Promise (<anonymous>)
    at t.value (apexcharts.min.js?3d1e:14)
    at eval (apexcharts.min.js?3d1e:14)
```

Its coming from the destroy() function of the lib and it seems that either "window" or "this.events" is null at the point of the call

## Explanation

- What is the behavior you expect?
No error when I switch the View before the chart finished loading

- What is happening instead?
An error is thrown in the console when switching the View before the chart finished loading

- What error message are you getting?
Uncaught (in promise) TypeError: Cannot read property 'removeEventListener' of null

**Comment by MichalNemec** [comment]
+1 have the same problem but in angular (ng-apexcharts)

**Comment by Hobbytowo** [comment]
I think it resolving also my bug on window resize in react:
> Unhandled Rejection (TypeError): t.w.globals.dom.baseEl is null

when can we expect releas with this fix?

**Comment by mattilind** [comment]
I am also getting the same error when the HTML element containing the chart is hidden during a data request and shown afterwards. However, this error does not seem to cause any problems with the functionality of the chart. The error started occurring after updating apexcharts from 3.19.2 to 3.20.0. I am unable to reproduce this in a CodePen.

**Comment by peteichuk** [comment]
This bug in apexcharts.js. [Issue](https://github.com/apexcharts/apexcharts.js/issues/1891)

**Comment by Hobbytowo** [comment]
Fixed in ^3.20.1, thank you :)
